package.json

```
@@ -29,7 +29,7 @@
   },
   "dependencies": {
     "@optimizely/js-sdk-logging": "^0.1.0",
-    "@optimizely/optimizely-sdk": "4.3.1",
+    "@optimizely/optimizely-sdk": "4.3.3",
```

I merged #75 , so we can make this a compatible-version dependency instead of an exact dependency:
```suggestion
    "@optimizely/optimizely-sdk": "^4.3.3",
```
I think this can be done with the `yarn add` command.